Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "wkt"
 description = "Rust read/write support for well-known text (WKT)"
-version = "0.9.2"
-authors = ["Corey Farwell <coreyf@rwell.org>", "Antoine Desbordes <antoine.desbordes@gmail.com>"]
+version = "0.10.0"
+authors = ["The Georust Developers <mods@georust.org>"]
```

Comment:
We could also drop this. The field has been optional for a while, and setting it is somewhat discouraged.

Reply:
@lnicola makes sense, want to make a PR?